### PR TITLE
P20-1101: Remove unsupported School Project OpenID integration

### PR DESCRIPTION
**Pull request by artem-vavilov**
This cleanup allows us to eliminate the dependency on the `omniauth-openid-connect` gem.

## Links
- JIRA ticket: [P20-1101](https://codedotorg.atlassian.net/browse/P20-1101)
- Original PR: https://github.com/code-dot-org/code-dot-org/pull/13649

### CODE REVIEW

**Review by carl-codeorg — APPROVED** [review]
LGTM. Thanks for cleaning this up!

**Review by wilkie — APPROVED** [review]
Looks good to me!

File: Gemfile [hunk 1]
```diff
@@ -287,6 +287,7 @@ gem 'addressable'
 gem 'bcrypt', '3.1.13'
 gem 'sshkit'
 gem 'validates_email_format_of'
+gem 'validate_url', '~> 1.0.15'
```
Comment (artem-vavilov) [inline Gemfile:290]: https://github.com/code-dot-org/code-dot-org/pull/60526/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fL640
https://github.com/code-dot-org/code-dot-org/blob/6cad4ce8d6eb32745010662d37ec8980b83beda5/dashboard/app/models/lti_course.rb#L32

Comment (wilkie) [reply Gemfile:290]: Very nice catch. How did you spot this?

Comment (artem-vavilov) [reply Gemfile:290]: https://drone.cdn-code.org/code-dot-org/code-dot-org/12961/1/6
```
[GEM]/activemodel-6.1.7.7/lib/active_model/validations/validates.rb:121:in `rescue in block in validates': Unknown validator: 'UrlValidator' (ArgumentError)
	from [CDO]/dashboard/app/models/lti_course.rb:32:in `<class:LtiCourse>'
```

File: Gemfile [hunk 2]
```diff
@@ -366,3 +366,4 @@ gem 'mailgun-ruby', '~>1.2.14'
 gem 'mailjet', '~> 1.7.3'
 
 gem "json-schema", "~> 4.3"
+gem 'json-jwt', '~> 1.15'
```
Comment (artem-vavilov) [inline Gemfile:368]: https://github.com/code-dot-org/code-dot-org/pull/60526/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fL634
https://github.com/code-dot-org/code-dot-org/blob/a9d709120d3acc12ec8213fbafd84106a1260c01/dashboard/app/controllers/lti_v1_controller.rb#L74

**Pull request merged**